openpype/pipeline/create/context.py

```diff
@@ -1185,8 +1184,8 @@ def reset_finalization(self):
         self._collection_shared_data = None
         self.refresh_thumbnails()
 
-    def reset_avalon_context(self):
-        """Give ability to reset avalon context.
+    def reset_current_context(self):
+        """Refresh current context.
```

Comment:
Name is much better, nice!

I did notice that the docstring below is actually incomplete.

> tool but that change is not propagated to

To where?